collectors/o365/lib/o365_mgmnt/index.js

```diff
@@ -36,13 +34,12 @@ function getO365ManagmentClient(creds) {
  * Office 365 Management API subscription/content.
  * {@link https://docs.microsoft.com/en-us/office/office-365-management-api/office-365-management-activity-api-reference#list-current-subscriptions Reference.}
  * 
+ * @param {O365Management} client - MSAzureObject for communicating with O365 management APIs
  * @returns {Promise}
  *
  */
-var _listSubscriptions = function(creds) {
```

Comment:
I think we can remove all the function down below bcause consumets can call client function directly, like `client.listSubscriptions(null)`.